test/images/README.md

```diff
@@ -0,0 +1,113 @@
+Docker daemon without requiring any authentication is not recommended, and thus, it must be
+configured with TLS to ensure that only authorised people can interact with it. For this, the
+following `powershell` script can be executed:
+
+```powershell
+    mkdir server
+    mkdir client\.docker
+    docker run --rm `
+      -e SERVER_NAME=$(hostname) `
+      -e IP_ADDRESSES=127.0.0.1,YOUR_LINUX_BUILD_NODE `
+      -v "$(pwd)\server:c:\programdata\docker" `
+      -v "$(pwd)\client\.docker:c:\users\containeradministrator\.docker" stefanscherer/dockertls-windows:1809
+    docker run --rm `
+      -e SERVER_NAME=$(hostname) `
+      -e IP_ADDRESSES=127.0.0.1,YOUR_LINUX_BUILD_NODE `
+      -v "c:\programdata\docker:c:\programdata\docker" `
+      -v "$env:USERPROFILE\.docker:c:\users\containeradministrator\.docker" stefanscherer/dockertls-windows:1809
+    # restart the Docker daemon.
+    Restart-Service docker
+```
+
+For more information about the above commands, you can check [here](https://hub.docker.com/r/stefanscherer/dockertls-windows/).
+
+A firewall rule to allow connections to the Docker daemon is necessary:
+
+```powershell
+    New-NetFirewallRule -DisplayName 'Docker SSL Inbound' -Profile @('Domain', 'Public', 'Private') -Direction Inbound -Action Allow -Protocol TCP -LocalPort 2376
+```
+
+The `ca.pem`, `cert.pem`, and `key.pem` files that can be found in `$env:USERPROFILE\.docker`
+will have to copied to the `~/.docker/` on the Linux build node. After all this, the Linux
+build node should be able to connect to the Windows build node:
+
+```bash
+    docker --tlsverify -H "$REMOTE_DOCKER_URL" version
+```
+
+For more information and troubleshooting about enabling Docker remote management, see
+[here](https://docs.microsoft.com/en-us/virtualization/windowscontainers/management/manage_remotehost)
+
+Finally, the Windows node must be able to push the built images to the desired registry. For
+dockerhub, the following command can be executed:
+
+```powershell
+    docker login -u your-awesome-username -p anAwesomerPassword
+```
+
+## Building images
+
+The images are built through `make`. Since some images (`busybox` ,`mounttest`, `test-webserver`)
+are used as a base for other images, it is recommended to built them first.
+
+An image can be built by simply running the command:
+
+```bash
+    make all WHAT=test-webserver
+```
+
+To build AND push an image, the following command can be used:
+
+```bash
+    make all-push WHAT=test-webserver
+```
+
+By default, the images will be tagged and pushed under the `gcr.io/kubernetes-e2e-test-images`
+registry. That can changed by running this command instead:
+
+```bash
+    REGISTRY=foo_registry make all-push WHAT=test-webserver
+```
+
+In order to also include Windows Container images into the final manifest lists, the
+`REMOTE_DOCKER_URL` argument will also have to be specified:
+
+```bash
+    REMOTE_DOCKER_URL=remote_docker_url REGISTRY=foo_registry make all-push WHAT=test-webserver
+```
```

Comment:
Please add the issue mentioned under known issue and workaround from - https://github.com/docker/for-linux/issues/396 to make doc complete

Reply:
Done, I've simply added the issue title and a link to it. All the information and solutions can be read on the issue, and if the issue will ever be reopened, or if there will be new information, it will probably be added there.

Reply:
thanks for adding that information @bclau